doc/radosgw/lua-scripting.rst

```diff
@@ -290,6 +290,10 @@ Request Fields
 +----------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
 | ``Request.User.Id``                                | string   | triggering user id                                           | no       | no        | no       |
 +----------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``Request.Trace``                                  | table    | info on trace                                                | no       | no        | yes      |
++----------------------------------------------------+----------+--------------------------------------------------------------+----------+-----------+----------+
+| ``Request.Trace.Enable``                           | boolean  | tracing is enabled                                           | no       | yes       | yes      |
```

Comment:
please explain what is the default value of that field.
IMO, should be derived from the global tracing configuration

Reply:
I think the default value can be true. it acts the same as if we fetch the global tracing configuration, and it is less confusing.
if the tracer is disabled - a noop span will be returned anyway.
if the tracer is enabled -  as well the Request.Trace.Enable will be true so an actual span will be started.

Reply:
ok. then please add to the doc the fact that the default value is "true". and also that it does not override the configured value, but only allows disabling for specific requests.

Reply:
that default makes sense for *writers* of `Request.Trace.Enable`, but are there cases where lua script just wants to *read* whether tracing is on or off?

Reply:
indeed

Reply:
fixed. thanks